devtools/conda-envs/openeye-examples.yaml

```diff
@@ -1,11 +1,12 @@
 name: openeye-examples
 channels:
+  - conda-forge/label/openff-interchange_rc
   - openeye
   - conda-forge
 dependencies:
     # Base depends
   - python
-  - pydantic =1
+  - pydantic
```

Comment:
For both packaging and testing yamls, should we set this to `pydantic >2`? Or is it redundant since it will be impossible to install this with a pydantic-1-compatible Interchange?

Reply:
* I think `pydantic >2`/`pydantic >=2` would be unwise since that bets on current builds being compatible with Pydantic v3+ out of the box, which I don't trust. `pydantic =2` should be fine, if a little redundant since the toolkit itself has no direct opinions on Pydantic compatibility
* Yes, installing v1 (install-time/packaging level) will be impossible with Interchange 0.4